scripts/deck_code.py

```
@@ -18,38 +18,154 @@
+    specials, normals = sorted(n for n in nrs if n >= SPECIAL_THRESHOLD), sorted(
+        n for n in nrs if n < SPECIAL_THRESHOLD)
+    b = bytearray()
+
+    for group in (specials, normals):
+        b.append(len(group) & 0xff)
         for n in group:
             v = n * 10
-            bytes_arr.append((v >> 16) & 0xff)
-            bytes_arr.append((v >> 8) & 0xff)
-            bytes_arr.append(v & 0xff)
+            if v > 0xFFFFFF: raise ValueError("ID exceeds 3 bytes")
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)